src/perform_step/adams_bashforth_moulton_perform_step.jl

```diff
@@ -574,14 +574,18 @@ end
     grid_points[3] = t
   end
 
-  cache.grid_points = copy(grid_points)
+  cache.grid_points .= grid_points
```

Comment:
why is there two sets of grid points?

Reply:
no need of the second one, I will remove one.

Reply:
Neither of them should be necessary though. They are the same array.